crates/y-sweet-worker/src/s3_store.rs

```diff
@@ -18,17 +19,23 @@ pub struct S3Store {
 
 impl S3Store {
     pub fn new(
+        endpoint: String,
```

Comment:
might be easier to just take the `S3Config` object directly? We will want to move both into `crates/y-sweet-core` eventually